src/app/myworkflows/sidebar-accordion/sidebar-accordion.component.html

```diff
@@ -1,38 +1,44 @@
               {{workflowObj?.repository + (workflowObj?.workflowName ? '/' + workflowObj?.workflowName : '')}}
             </a>
-          </div>
-        </div>
-        <div *ngIf="orgObj?.published.length == 0" class="text-center">
-          <p>No published workflows.</p>
-        </div>
-      </tab>
-      <tab heading="Unpublished" [active]="orgObj.activeTab === 'unpublished'">
-        <div *ngFor="let workflowObj of orgObj?.unpublished">
-          <div class="panel-container-label" [ngClass]="{selected: (workflowId$ | async) === workflowObj?.id}">
+          </mat-list-item>
+          <mat-list-item *ngIf="!orgObj || orgObj?.published.length === 0">
+            <p class="text-center">No published workflows.</p>
+          </mat-list-item>
+        </mat-list>
+      </mat-tab>
+      <mat-tab label="Unpublished">
+        <mat-list class="org-accordion-list" >
+          <mat-list-item *ngFor="let workflowObj of orgObj?.unpublished" [ngClass]="{selected: (workflowId$ | async) === workflowObj?.id}">
             <a class="no-wrap" title="{{workflowObj?.repository + (workflowObj?.workflowName ? '/' + workflowObj?.workflowName : '')}}" [routerLink]="'/my-workflows/' + workflowObj.full_workflow_path">
               {{workflowObj?.repository + (workflowObj?.workflowName ? '/' + workflowObj?.workflowName : '')}}
             </a>
-          </div>
-        </div>
-        <div *ngIf="!orgObj || orgObj?.unpublished.length === 0" class="text-center">
-          <p>No unpublished workflows.</p>
-        </div>
-      </tab>
-    </tabset>
-    <span *ngIf="!orgObj?.sourceControl.startsWith('dockstore.org')">
-      <app-refresh-workflow-organization class="pull-right" [organization]="orgObj?.organization"></app-refresh-workflow-organization>
-    </span>
-    <button type="button" class="glyphicon glyphicon-plus btn btn-primary btn-xs pull-right add-container-ns" (click)="setRegisterEntryModalInfo(orgObj?.organization + '/new_workflow'); showRegisterEntryModal()">
-    </button>
-  </accordion-group>
-</accordion>
+          </mat-list-item>
+          <mat-list-item *ngIf="!orgObj || orgObj?.unpublished.length === 0">
+            <p class="text-center">No unpublished workflows.</p>
+          </mat-list-item>
+        </mat-list>
+      </mat-tab>
+    </mat-tab-group>
+    <mat-action-row>
+      <span *ngIf="!orgObj?.sourceControl.startsWith('dockstore.org')">
+        <app-refresh-workflow-organization [organization]="orgObj?.organization"></app-refresh-workflow-organization>
+      </span>
+      <button mat-mini-fab color="accent" matTooltip="Add new workflow" [matTooltipShowDelay]="1000" [matTooltipPosition]="'after'" type="button" (click)="setRegisterEntryModalInfo(orgObj?.organization + '/new_workflow'); showRegisterEntryModal()">
```

Comment:
These buttons are too close to each, IMHO. I think the edges are actually touching each other.